irods/test/query_test.py

```diff
@@ -1019,6 +1021,33 @@ def test_not_like_operator__issue_443(self):
         )
         self.assertEqual(3, len(list(query)))
 
+    def test_skipping_columns_in_genquery1_results__issue_755(self):
+        unneeded_columns = {Collection.map_id, DataObject.map_id, DataObject.status, DataObject.type, DataObject.collection_id}
+        unneeded_columns_D = unneeded_columns & set(DataObject._columns)
+        unneeded_columns_C = unneeded_columns & set(Collection._columns)
```

Comment:
The use of the word "unneeded" makes it seem like they aren't the main focus of the test.

How about this ... `columns_to_negate`?

Reply:
Done!